addons/binding/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/LGWebOSBindingConstants.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.lgwebos;
+
+import java.util.Set;
+
+import org.eclipse.jdt.annotation.NonNull;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+import com.google.common.collect.ImmutableSet;
+
+/**
+ * This class defines common constants, which are used across the whole binding.
+ *
+ * @author Sebastian Prehn - Initial contribution
+ */
+public class LGWebOSBindingConstants {
+
+    public static final @NonNull String BINDING_ID = "lgwebos";
```

Comment:
Otherwise I get a warning on the next line regarding Null safety where this constant is used:
```public static final ThingTypeUID THING_TYPE_WEBOSTV = new ThingTypeUID(BINDING_ID, "WebOSTV");```

Reply:
Given this is so obvious @NonNull already I would personally prefer to leave out the annotation, @triller-telekom wdyt?

Reply:
In out [coding guidelines in point 12](https://www.eclipse.org/smarthome/documentation/development/guidelines.html#a-code-style) we agreed that IF one chooses to use null annotations he has to use @NonNullByDefault on the class level and only @NonNull on variables/method parameters if they can be null.

Regarding the warning/error in the IDE, we are still consulting the setting which we will use for the null analysis, see our discussion here: https://github.com/eclipse/smarthome/issues/4321